build/README.md

```diff
@@ -229,3 +229,9 @@ Pick a username and create an account:
 
 Then go to https://localhost:8080. You'll have to ignore the HTTPS certificate
 warning.
+
+## Updgrading Database Schemas
+
+All Schemas related files are in `deploy/db-schemas` directory, any changes you wish to make to the Database schema should be done in their respective Database folders. The files are applied in steps from 1 to N in the direction specified in the file's suffix.
+
+For the first ever run during the CRDB cluster initilization the db-manager will run once to bootstrap and bring up the database up to date. In order to upgrade existing clusters you will need to update the `desired_xyz_db_version` field in `main.jsonnet`, delete the existing db-manager job in your k8s cluster and redeploy the newly configured db-manager. It should automatically up/down grade your database schema to your desired version.
```

Comment:
What would happen if a new participant started up their Kubernetes cluster for the first time when connecting to an existing CockroachDB cluster currently hosted by other participants?

Basically, I have these use cases in mind:
1) An individual user wants to start a brand new CRDB cluster + Kubernetes cluster
2) A first-in-region organization wants to start a brand new CRDB cluster (in a brand new Kubernetes cluster), and the CRDB cluster will be later be joined by other participants
3) An organization wants to start a brand new Kubernetes cluster that joins an existing CRDB cluster
4) An organization wants to migrate the schema of an existing CRDB cluster in an existing Kubernetes cluster

I want to make sure migration will not be run for use case 3 (thereby potentially breaking the system for all other participants), even if the joining organization accidentally had `shouldInit=true` or some other easy-to-miss incorrect configuration.

Reply:
I'll add an assert to make sure that only shouldInit is true or Joining Cluster is populated but never both. This will prevent use case 2,3,4 to not run the db-manager